meecrowave-core/src/main/java/org/apache/meecrowave/Meecrowave.java

```diff
@@ -116,7 +116,17 @@
 import static java.util.stream.Collectors.toList;
 
 public class Meecrowave implements AutoCloseable {
-    private final Builder configuration;
+    
+	private static final String MEECROWAVE_PROPERTIES = "meecrowave.properties";
```

Comment:
i'd prefer to replace the get/set pattern with a class where we have a single String var and keep it inlined, it is more readable IMHO